docs/collecting-data/code-generation/commands/index.md

```diff
@@ -40,7 +40,7 @@ Checks for latest version updates in Data Structures and Event Specifications.
 
 ### `snowtype patch`
 
-Adds new Data Structures and Event Specifications in the `snowtype.config.json` file without needing to modify the file by hand.
+Adds new Data Structures and Event Specifications in the `snowtype.config.json`, `snowtype.config.js`, or `snowtype.config.ts` file without needing to modify the file by hand.
```

Comment:
`patch` only works on JSON right ?

Reply:
You're absolutely right, that should stay only for JSON. I'll sort it, thanks!